spring-integration-azure/src/main/java/com/microsoft/azure/spring/integration/eventhub/HostnameHelper.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.spring.integration.eventhub;
+
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class HostnameHelper {
+
+    private static final String UNKNOWN_HOST_NAME = "Unknown-HostName";
+
+    private static final String HOST_NAME = getNetworkHostname();
+
+    private static String getNetworkHostname() {
```

Comment:
I think `getHostName` is good enough, `network` here is confusing.

Reply:
As there is public method outside take this name, I add details information for where the hostname from.